simbad/parsers/anode_parser.py

```diff
@@ -33,3 +33,9 @@ def parse(self):
                     self.peak_height = fields[4]
                     self.nearest_atom = fields[-1]
                 line = f.readline()
+
+    def summary(self):
+        pass
+
+    def check_input(self):
+        pass
```

Comment:
Consider removing the methods `check_input` and `summary` from the base class `simbad.parsers._Parser`. You don't seem to use them in any child class as you are just overriding them with `pass`. If there's an isolated case where you actually use it in a child parser class, maybe just add the method to that particular child class instead.